tests/test_bar/path/xrandr

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+cat << EOF
+Screen 0: minimum 8 x 8, current 1920 x 1200, maximum 16384 x 16384
```

Comment:
If you have working multimonitor configuration it is better to place it here instead of what currently this file contains. I have only one monitor: DVI-I-1, VGA-0 is a copy-paste with two edits (s/DVI-I-1/VGA-0/ and s/1200+0/1200+1/).